packages/shared/src/index.ts

```diff
@@ -36,4 +35,3 @@ export * from './object';
 export * from './logger';
 export { createWorkerTimers } from './workerTimers';
 export { DEV_BROWSER_JWT_KEY, extractDevBrowserJWTFromURL, setDevBrowserJWTInURL } from './devBrowser';
-export { fastDeepMergeAndKeep, fastDeepMergeAndReplace } from './utils/fastDeepMerge';
```

Comment:
This would require the same treatment as https://github.com/clerk/javascript/pull/4972, right?

Reply:
Another duped export

![Screenshot 2025-01-23 at 9 23 01 AM](https://github.com/user-attachments/assets/292540fc-07cf-4af9-ab35-aeb0bf06bd96)